OasisDEX/oasis-borrow
# Scaffolding: Aave V3 L1 positions on positions overview

# [Owner page - aavev3 L1](https://app.shortcut.com/oazo-apps/story/7739/owner-page-aavev3)

Untested for Aave V3 positions.  Should cause no regressions for Aave V2 and Maker positions.

<please insert a shortcut link above>
  
## Changes 👷‍♀️

- Creates a new observable injecting down the Aave V3 things.
- The positions overview table support long position titles.
- `readPositionCreatedEvents` takes an optional filter for the protocol for the event.
  
## How to test 🧪

- Ensure that Aave V2 (DPM and DS), and Maker positions load and display correctly.
- Make sure page doesn't break.
- Don't worry about Aave V3.
